src/tsdb/prw.pb-c.c

```diff
@@ -0,0 +1,380 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
```

Comment:
Since we're just including the .c file here I presume that we think this prw proto is relatively stable and won't need to be regenerated, right? That's OK, but I think it would be good to include a reference to the .proto and how this file was generated, since those things are not part of any Makefile or similar here now.

In the future I think we could have support in Acton for generating stuff like this.. but one thing at a time ;)